azure-mgmt-network/src/main/java/com/microsoft/azure/management/network/FlowLogSettings.java

```diff
@@ -0,0 +1,134 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+package com.microsoft.azure.management.network;
+
+import com.microsoft.azure.management.apigeneration.Beta;
+import com.microsoft.azure.management.apigeneration.Fluent;
+import com.microsoft.azure.management.network.implementation.FlowLogInformationInner;
+import com.microsoft.azure.management.resources.fluentcore.arm.models.HasParent;
+import com.microsoft.azure.management.resources.fluentcore.model.Appliable;
+import com.microsoft.azure.management.resources.fluentcore.model.HasInner;
+import com.microsoft.azure.management.resources.fluentcore.model.Refreshable;
+import com.microsoft.azure.management.resources.fluentcore.model.Updatable;
+
+/**
+ * Client-side representation of the configuration of flow log, associated with network watcher and an Azure resource.
+ */
+@Fluent
+@Beta
+public interface FlowLogSettings extends
+        HasParent<NetworkWatcher>,
+        HasInner<FlowLogInformationInner>,
+        Updatable<FlowLogSettings.Update>,
+        Refreshable<FlowLogSettings> {
+    /**
+     * Get the ID of the resource to configure for flow logging.
+     *
+     * @return the targetResourceId value
+     */
+    String targetResourceId();
+
+    /**
+     * Get the ID of the storage account which is used to store the flow log.
+     *
+     * @return the storageId value
```

Comment:
we could keep only comment for `return` as `the id of the storage account used to store the flow log ` and remove the method comment